examples/research_projects/bertology/run_prune_gpt.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python3
+""" This script is adapted from the Bertology pruning code (https://github.com/huggingface/transformers/blob/783d7d2629e97c5f0c5f9ef01b8c66410275c204/examples/research_projects/bertology/run_bertology.py)
+to prune GPT-like models. The author is @altsoph.
+"""
+
+from __future__ import absolute_import, division, print_function

```

Comment:
This is just for python 2 compatibility, no? We're on 3.6 min so this can be removed.

Reply:
Yes, thanks!